src/datasources/db/v2/postgres-database.module.ts

```diff
@@ -1,8 +1,11 @@
 import { Module } from '@nestjs/common';
 import { PostgresDatabaseService } from '@/datasources/db/v2/postgres-database.service';
+import { DatabaseMigrator } from '@/datasources/db/v2/database-migrator.service';
+import { ConfigModule } from '@nestjs/config';
```

Comment:
As before regarding usage of `@/config/configuration.module`.

Reply:
Unfortunately, Since we need `env` variables, it's not possible to use `configuration.module`.